backend/fms_core/models/individual.py

```diff
@@ -92,6 +95,12 @@ def add_error(field: str, error: str):
         if self.mother_id is not None and self.mother.sex == self.SEX_MALE:
             add_error("mother", "Mother cannot be of male sex.")
 
+        if self.is_generic and self.name[:len(self.GENERIC_INDIVIDUAL_PREFIX)] != self.GENERIC_INDIVIDUAL_PREFIX:
+            add_error("is_generic", f"'{self.GENERIC_INDIVIDUAL_PREFIX}' name prefix not used for generic individual.")
+
+        if not self.is_generic and self.name[:len(self.GENERIC_INDIVIDUAL_PREFIX)] == self.GENERIC_INDIVIDUAL_PREFIX:
+            add_error("is_generic", f"'{self.GENERIC_INDIVIDUAL_PREFIX}' name prefix cannot be used for non generic individuals.")
+
```

Comment:
Maybe we can reuse certain expressions such as
```python
has_generic_individual_prefix = self.name[:len(self.GENERIC_INDIVIDUAL_PREFIX)] == self.GENERIC_INDIVIDUAL_PREFIX
```

Reply:
I could store that somewhere but i would rather this stays there and people rely on the attribute ... it's a maybe i'll think about it.